plugins/module_utils/utils.py

```diff
@@ -146,3 +146,13 @@ def check_if_all_elements_are_none(values):
     :return: True if all elements are None, False otherwise. -> bool
     """
     return all(value is None for value in values)
+
+
+def snake_to_camel(snake_str):
+    if snake_str is not None and "_" in snake_str:
+        exceptions = ["ip"]
```

Comment:
what is this exception for? should we add explanation for it?

Reply:
Made changes as discussed